src/MongoDB.Driver/Linq/Translators/AggregateLanguageTranslator.cs

```diff
@@ -1276,5 +1314,19 @@ private bool TryTranslateSumResultOperator(PipelineExpression node, out BsonValu
             result = null;
             return false;
         }
+
+        private bool TryTranslateViaToString(MethodCallExpression node, BsonValue field, out BsonValue result)
```

Comment:
I've added the ability to specify not only decimal.ToString(), but all supported types for this command